### PR TITLE
add minutes from ops-iam 2022-10-06

### PR DESCRIPTION
**Pull request by fkr**
Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

### CODE REVIEW

File: ops-iam/20221006.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# 2022-10-06
+## Participants  use of github handle is preferred
+* @fkr
+* @jnull
+* @lossack
+@JuanPTM
```
Comment (matfechner) [inline ops-iam/20221006.md:6]: should here be  a asterisk or space ?

**Review by matfechner — COMMENTED** [review]
LGTM with the one comment

File: ops-iam/20221006.md (hunk 1 above)
Comment (fkr) [reply ops-iam/20221006.md]: thanks for spotting!

**Review by matfechner — APPROVED** [review]
LGTM

**Pull request merged**